nixos/modules/flyingcircus/roles/nfs.nix

```diff
@@ -90,5 +93,30 @@ in
         ${pkgs.nfs-utils}/bin/exportfs -ra
       '';
     })
+
+    (mkIf (boxServer ? address) (
+      let
+        humanUsers =
+          (filterAttrs (n: v: v.isNormalUser && v.group == "users")
+          config.users.users);
+      in
+      {
+        boot.supportedFilesystems = [ "nfs4" ];
```

Comment:
Should this really be dependent on whether there is a box server or not? Does this mean we have to reboot all VMs when we enable/disable box servers? Also, isn't this the client code that manages nfs mounts anyway? Seems superfluous to me. Or rather: I don't quite agree with the existing pattern and maybe we haven't seen this as a problem yet even though it might be.